content/shmem_team_alltoall_nbi.tex

```diff
@@ -0,0 +1,88 @@
+\apisummary{
+    Initiates a nonblocking transfer of fixed-size blocks of data from each
+    \ac{PE} in a team to all \acp{PE} in the team.
+}
+
```

Comment:
Since nonblocking collectives are a larger change to the spec, the working group would like to discuss them separately and have them in a different PR.